lib/CMakeLists.txt

```diff
@@ -41,29 +41,31 @@ if (WIN32)
     set(ofx_HEADERS ${ofx_HEADERS} win32.hh)
 endif ()
 
-add_library(ofx ${ofx_SRCS} ${ofx_HEADERS})
```

Comment:
Sorry, this change on Linux ends up in a file `libLibOFX.so.7`. The target name used in `add_library` shouldn't include the "lib..." prefix.

Reply:
Fair enough. I'll fix it and let you know.

Reply:
Just wondering what should the target name be? `LibOFX::ofx`, `libofx::ofx` or `ofx::ofx`? I feel like the latter is too generic.

Reply:
OK, I managed to get it to be `libofx::libofx` and keep the `libofx` library name at the same time